libs/styles/styles.css

```diff
@@ -1,134 +1,586 @@
+  --sn-drop-shadow-level4-default-color: var(--s2-drop-shadow-color-400); /* NARRATIVE */
+
+  /*** S2 Corner Radius ***/
+  --s2-corner-radius-100: 4px;
+  --s2-corner-radius-300: 6px;
+  --s2-corner-radius-500: 8px;
+  --sn-corner-radius-800: 12px; /* NARRATIVE */
+  --s2-corner-radius-900: 16px; /* S2 missing */
+  --sn-corner-radius-1000: 20px; /* NARRATIVE */
+  --sn-corner-radius-1100: 24px; /* NARRATIVE */
+  --s2-corner-radius-2000: 32px ; /* S2 missing */
+
+  /*** S2 Spacing Spec ***/
+  --s2-spacing-75: 4px;
+  --s2-spacing-100: 8px;
+  --s2-spacing-200: 12px;
+  --s2-spacing-300: 16px;
+  --s2-spacing-400: 24px;
+  --s2-spacing-500: 32px;
+  --s2-spacing-600: 40px;
+  --s2-spacing-700: 48px;
+  --s2-spacing-800: 64px;
+  --s2-spacing-900: 80px;
+  --s2-spacing-1000: 96px;
+  --sn-spacing-1100: 104px; /* NARRATIVE */
+
+  /*** S2 Type Size Spec ***/
+  --s2-font-size-75: 12px;
+  --s2-font-size-100: 14px;
+  --s2-font-size-200: 16px;
+  --s2-font-size-300: 18px;
+  --s2-font-size-400: 20px;
+  --s2-font-size-500: 22px;
+  --s2-font-size-600: 25px;
+  --s2-font-size-700: 28px;
+  --s2-font-size-800: 32px;
+  --s2-font-size-900: 36px;
+  --s2-font-size-1000: 40px;
+  --s2-font-size-1100: 45px;
+  --s2-font-size-1200: 50px; /* should be 51px */
+  --s2-font-size-1300: 60px; /* should be 58px */
+  --s2-font-size-1400: 65px; /* missing */
+  --s2-font-size-1500: 73px; /* missing */
+
+  /*** S2 Font Weight ***/
+  --s2-regular-font-weight: 400;
+  --s2-medium-font-weight: 500;
+  --s2-bold-font-weight: 700;
+  --s2-extra-bold-font-weight: 800;
+  --s2-black-font-weight: 900;
+
+  /* S2 Workflow Icon Size */
+  --s2-workflow-icon-size-50: 14px;
+  --s2-workflow-icon-size-75: 16px;
+  --s2-workflow-icon-size-100: 20px;
+  --s2-workflow-icon-size-200: 22px;
+  --s2-workflow-icon-size-300: 26px;
+  /* S2 Chevron Icon Size */
+  --s2-chevron-icon-size-100: 10px;
+  --s2-chevron-icon-size-200: 12px;
+  --s2-chevron-icon-size-300: 14px;
+  --s2-chevron-icon-size-400: 16px;
+
+  /* icon sizes (milo original) */
+  --icon-size-xxl: 80px;
+  --icon-size-xl: 64px;
+  --icon-size-l: 56px;
+  --icon-size-m: 40px;
+  --icon-size-s: 32px;
+  --icon-size-xs: 24px;
+  --icon-size-xxs: 16px;
 
-  /* Fonts */
-  --body-font-family: 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+  /* S2 Grid sizes */
+  --grid-container-width: 100%; /* previously 83.4% */
+  --grid-column-width: calc(var(--grid-container-width) / 8);
+  --grid-margins-width: 16px; /* previously calc((100vw - var(--grid-container-width)) / 2) */
+  /* milo originals */
+  --grid-margins-width-10: calc((100vw - (var(--grid-column-width) * 10)) / 2);
+  --grid-margins-width-8: calc((100vw - (var(--grid-column-width) * 8)) / 2);
+  --grid-margins-width-6: calc((100vw - (var(--grid-column-width) * 6)) / 2);
+
+  @media (min-width: 768px) { /* Tablet */
```

Comment:
Where are we getting the `768` value from? Is it from somewhere in Spectrum? Currently, we consider `900` to be the shift from mobile to desktop, we should get alignment on the value. On resolutions between 768-900, we would get mobile styles from the blocks, but tablet styles from this file. Similar question for 1280, 1680.

Reply:
Yes these are the new media queries specified by S2. They can be found in these specs [here. ](https://www.figma.com/design/9R2PrV8kBaICcz4xLX1rZQ/Spectrum-Narrative-Exploration?node-id=1379-37572&m=dev)